tests/test_fpca.py

```diff
@@ -58,7 +60,9 @@ def test_basis_fpca_fit_result(self):
         fd_basis = fd_data.to_basis(basis)
 
         fpca = FPCABasis(n_components=n_components,
-                         regularization_parameter=1e5)
+                         regularization=TikhonovRegularization(
```

Comment:
Is the Tikonov regularization equivalent to the left centered penalty that I implemented earlier?